runner/runner.pyproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
```

Comment:
why is this file added?

Reply:
If you don't want to open as VS project then just don't and no impact. I have found this much more productive way to manage python projects